projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1603,16 +1627,277 @@ public static function wrap_message_in_html_tags( $body ) {
 			str_replace(
 				"\t",
 				'',
-				'<!doctype html>
-				<html xmlns="http://www.w3.org/1999/xhtml">
-				<body>
-
-				%s
-
-				</body>
-				</html>'
+				apply_filters(
+					'jetpack_forms_respone_email_template',
+					'
+					<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+					<html xmlns="http://www.w3.org/1999/xhtml">
+						<head>
+							<title>Jetpack Forms Response</title>
```

Comment:
Should this be translatable?

Reply:
👍  Yes, I think so, but only the "Response" part, not the "product name" Jetpack Forms.

Reply:
I have to make the entire string translatable as some languages will put the word "Responses" in front of "Jetpack Forms". For example, In Irish it would be "Freagraí Jetpack Forms" or perhaps "Jetpack Forms: Freagraí".